x-pack/plugins/cases/server/routes/api/cases/get_case.ts

```diff
@@ -26,53 +27,58 @@ const params = {
   }),
 };
 
-export const getCaseRoute = createCasesRoute({
-  method: 'get',
-  path: CASE_DETAILS_URL,
-  params,
-  routerOptions: {
-    access: 'public',
-    summary: `Get a case`,
-    tags: ['oas-tag:cases'],
-  },
-  handler: async ({ context, request, response, logger, kibanaVersion }) => {
-    try {
-      const isIncludeCommentsParamProvidedByTheUser =
-        request.url.searchParams.has('includeComments');
+export const getCaseRoute = ({ isServerless }: { isServerless?: boolean }) =>
```

Comment:
View it with hidden whitespaces. I just wrapped the code in a function.

Reply:
`resolveCaseRoute` also accepts `includeComments` and since these two functions share the `params` definition, vscode throws a deprecation warning there.

<img width="692" alt="Screenshot 2024-10-31 at 12 14 20" src="https://github.com/user-attachments/assets/ea6bd061-d4f7-4c04-81e8-91b92bfbab67">

Should we do something there too? I see the `use_get_case` hook needs it for example.

Reply:
The resolve API is already an internal API so it is fine if we do not take it into account in serverless.